src/api/storage.py

```diff
@@ -205,11 +266,20 @@ def sliding_window_train(self, sliding_window, prior=np.array([])):
                     else:
                         # Use previous sample to shift window
                         self._data.loc[index][col_labels[2:]
-                                              ] = list(prev_sample.iloc[-2*sliding_window:-2])
+                                              ] = list(
+                                                  prev_sample.iloc[
+                                                      -2*sliding_window:-2
+                                                  ]
+                        )
 
-                        # Use previous target to update first 2 positions in window
+                        # Use previous target to update first 2 positions
                         self._data.loc[index][col_labels[0:2]
-                                              ] = list(self._labels_arousal.loc[prev_sample_index]) + list(self._labels_valence.loc[prev_sample_index])
+                                              ] = list(
+                                                  self._labels_arousal.loc[
+                                                      prev_sample_index
+                                                  ]
+                        ) + \
+                            list(self._labels_valence.loc[prev_sample_index])
```

Comment:
I would put this last `list` thing on the same line, it doesn't get shorter at the moment. This exact thing happens in about one more place I believe.